docs/getting-started/quickstart-sms.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: quickstart-managed-scans
+append_help_link: true
+title: "Quickstart: Managed scans"
```

Comment:
I think in the blog and external communication we have referred to this as “Semgrep Managed Scanning” not “Semgrep Managed Scans” — but I see in the docs we use the latter. Did we ever standardize on one?

Reply:
yep, we standardized on [Managed Scans in the product](https://semgrepinc.slack.com/archives/C05J83YQXDG/p1720548816624589).

Some notes:

- You will notice the docs URL is managed-scanning, because it wasn't standardized at the time.
- The reasons why we went with Managed Scans were ultimately because
    - The page name (h1) in the product was **already** Managed scans
    - If we used both "Managed Scanning" and "Managed Scans" it would beg the question every time: "when do I use X or Y"? Whereas with Managed Scans you have the proper noun feature name and then a single managed scan is just a common noun.